### ISSUE TITLE
Report buttons not sending an output to wifi components

### ISSUE THREAD

**Issue by SeaverBeaver** [Regalis11/Barotrauma#7542]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
The report buttons, (Fire, Leak, Broken Devices, etc) don't send a detectable output to chat when pressed. You can see the chat message pop up, but WiFi components cant detect them, so you can't do anything with them.

**Steps To Reproduce**
Open the submarine editor.
-Wire a linked-to-chat WiFi component to a text display.
-Type something in radio chat, watch it appear on the text display.
-Press one of the report buttons, watch the message appear in chat, but the text display not show the message.

**Version**
Barotrauma v0.15.13.0 (the release version)

**Additional information**
I encountered this when attempting to make an intruder alert that would trigger when the "Report Intruders" button was pressed, I figured making a system that could function based off off of the incredibly convenient report buttons could save precious time in an emergency.

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/pull/3861/commits/04f81ce0bc7a77cf973b96fa31ccfd2a165e427a

**Comment by ekqvist** [comment]
Tested in singleplayer and multiplayer. Report messages seem to appear on the text displays (i.e. they are sent to the wifi components) now.

However, I did notice (at least in multiplayer, but could be an issue in singleplayer too) that sometimes the report message is different in chat and on the text display. Not sure if it's anything to address, but just noting this down. (IIRC there's logic that only gets the text for the message after the order message is received.)

Moving back to the Team 2 pipeline for you to determine what to do with this.

**Comment by ekqvist** [comment]
Tested again after the latest commit. The messages seem to be identical now in chat and on the displays. However, I noticed while also testing Regalis11/Barotrauma-development#1602 that after another player joins the lobby (while the first already has a game started) the report messages stop appearing the chat of the reporting player. They still appear in the text displays and in the chat of the second player in the lobby. But the player reporting them doesn't get them in their chat. It seems to fix itself after a while, after spamming the reports for some time.
